### PR TITLE
Update prettier

### PR DESCRIPTION
**Pull request by iAmWillShepherd**
New version of [Prettier](https://prettier.io/blog/2018/01/10/1.10.0.html)

- [x] Update Prettier
- [x] Check if Prettier plugins need updates
- [x] Make sure everything works
- [x] Apply new formatting

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,32 +5,43 @@
   },
   "description": "GitHub Desktop build dependencies",
   "scripts": {
-    "cli": "ts-node --require ./app/test/globals.ts --require ./app/src/cli/dev-commands-global.js app/src/cli/main.ts",
-    "test:integration": "cross-env TEST_ENV=1 ELECTRON_NO_ATTACH_CONSOLE=1 xvfb-maybe mocha -t 30000 --require ts-node/register app/test/integration/*.ts",
-    "test:unit": "cross-env GIT_AUTHOR_NAME=\"Joe Bloggs\" GIT_AUTHOR_EMAIL=\"joe.bloggs@somewhere.com\" GIT_COMMITTER_NAME=\"Joe Bloggs\" GIT_COMMITTER_EMAIL=\"joe.bloggs@somewhere.com\" TEST_ENV=1 ELECTRON_NO_ATTACH_CONSOLE=1 xvfb-maybe electron-mocha -t 10000 --renderer --require ts-node/register --require ./app/test/globals.ts app/test/unit/*.{ts,tsx} app/test/unit/**/*.{ts,tsx}",
+    "cli":
```
Comment (shiftkey) [inline package.json]: Let's not reformat this whole `package.json` file for now

Comment (iAmWillShepherd) [reply package.json]: TBH, that was not intentional, it reformatted on save as soon as I updated prettier. I wasn't expecting it to touch a JSON file.

Comment (j-f1) [reply package.json]: There’s an open issue at https://github.com/prettier/prettier/issues/3671 to ignore `package{,-lock}.json` by default.

Comment (shiftkey) [reply package.json]: @iAmWillShepherd all good, it's just a pet peeve of mine with prettier that @j-f1 can attest to

**Pull request merged**